scripts/SMCodeGen/dataModels/stateMachineAppModel.json

```diff
@@ -697,7 +697,6 @@
                             {
                                 "Name":"TAU_REQUEST_FROM_UE",
                                 "Actions":[
-                                    "PROCESS_TAU_REQUEST",
```

Comment:
which is the state where TAU leads to MBReq on S11?

Reply:
The TAU that we support currently is periodic TAU..We don't support tau with active flag as of now.